src/pages/TermsOfUsePage.tsx

```diff
@@ -1,17 +1,1106 @@
 import styled, { AnyStyledComponent } from 'styled-components';
 
-import { articleMixins } from '@/styles/articleMixins';
```

Comment:
Does the styling in articleMixins no longer apply?

Reply:
copy pasta-ed from a fork X____X